GETTING_STARTED.md

```diff
@@ -0,0 +1,602 @@
+# Getting started with kitchen-terraform
+
+Hello!  This tutorial will walk you through setting up a terraform config to spin up an AWS EC2 instance using Inspec and kitchen-terraform.
+
+Our Tools
+* [Terraform](https://www.terraform.io/)
+* [Test Kitchen](https://github.com/test-kitchen/test-kitchen)
+* [InSpec](https://github.com/chef/inspec)
+* [kitchen-terraform](https://github.com/newcontext/kitchen-terraform)
+
+Make sure you have the following prerequisites for this tutorial
+
+Prerequisites
+* An AWS Account
+* An AWS Access Key ID
+* An AWS Secret Key
+* An AWS Keypair - you will need the private key of this pair somewhere on your workstation
+* Terraform installed
+* Bundler installed
+* Ruby 2.3.1 installed and used (use your favorite ruby version manager, I used [rvm](https://rvm.io/) when creating this tutorial)
+
+So let's start building this config from scratch!
+
+## Setting up our development environment
+
+First, let's create a new directory for our config
+
+```
+  $ mkdir tf_aws_cluster
+```
+
+And cd into that directory
+
+```
+  $ cd tf_aws_cluster
+```
+
+Now, create some skeleton terraform files
+
+```
+  $ touch main.tf variables.tf output.tf testing.tfvars
+```
+
+Then create our test kitchen scaffolding
+
+```
+  $ kitchen init
+```
+
+And a Gemfile
+
+```
+  $ touch Gemfile
+```
+
+Open up that Gemfile using your favorite text editor
+
+```
+  $ vim Gemfile
+```
+
+And add in the kitchen-terraform gem like so
+
+Gemfile
+```
+  gem 'test-kitchen'
+  gem 'kitchen-terraform'
+```
+
+Then close and save the file, now run bundler to install these gems
+
+```
+  $ bundle install
+```
+
+(NOTE: You can install test-kitchen and kitchen-terraform however you prefer, I am using the Gemfile method for simplicity's sake, but feel free to use whatever method works for you and your work)
+
+## Setting up Test Kitchen
+
+Now let's set up Test Kitchen.  If you have used Test Kitchen before, setting it up with kitchen-terraform is a bit more complicated than other setups, but still doable!
```

Comment:
This comment is subjective; there are lots of complicated Test Kitchen plugins. :laughing: